boards/arm/nucleo_f429zi/doc/nucleof429zi.rst

```diff
@@ -0,0 +1,181 @@
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F429ZI can be found here:
+       - `STM32F429ZI on www.st.com`_
+       - `STM32F429 reference manual`_
+       - `STM32F429 datasheet`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_f249zi board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | Ethernet                            |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/nucleo_f429zi/nucleo_f429zi_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo F429ZI Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+Available pins:
+---------------
+.. image:: img/nucleo_f429zi_cn8.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F429ZI ZIO connectors (left)
+.. image:: img/nucleo_f429zi_cn7.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F429ZI ZIO connectors (right)
+.. image:: img/nucleo_f429zi_cn11.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F429ZI Morpho connectors (left)
+.. image:: img/nucleo_f429zi_cn12.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F429ZI Morpho connectors (right)
+
+For mode details please refer to `STM32 Nucleo-144 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_3_TX : PD8
+- UART_3_RX : PD9
+- PWM_2_CH1 : PA0
+- USER_PB : PC13
+- LD1 : PB0
+- LD2 : PB7
+- LD3 : PB14
+
+System Clock
+------------
+
+Nucleo F429ZI System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at 180MHz,
```

Comment:
as well as by the main...
By default the System clock is driven by the PLL clock...